Mathlib/Logic/Equiv/LocalEquiv.lean

```diff
@@ -24,7 +24,7 @@ namespace Tactic.MfldSetTac
 in one another. -/
 elab (name := mfldSetTac) "mfld_set_tac" : tactic => withMainContext do
   let g ← getMainGoal
-  let goalTy := (← instantiateMVars (← getMVarDecl g).type).getAppFnArgs
+  let goalTy := (← instantiateMVars (← MVarId.getDecl g).type).getAppFnArgs
```

Comment:
```suggestion
  let goalTy := (← instantiateMVars (← g.getDecl).type).getAppFnArgs
```
and elsewhere.

Reply:
ah the Haskell part of my brain has not learned yet that there is dot-notation, thanks.

Reply:
that's why they were deprecated in the first place! Otherwise it's just a slightly longer way to write the same thing.